src/test/rgw/CMakeLists.txt

```diff
@@ -170,3 +170,6 @@ add_executable(unittest_rgw_kms test_rgw_kms.cc)
 add_ceph_unittest(unittest_rgw_kms)
 
 target_link_libraries(unittest_rgw_kms ${rgw_libs})
+
+add_executable(ceph_test_rgw_gc_log test_rgw_gc_log.cc $<TARGET_OBJECTS:unit-main>)
+target_link_libraries(ceph_test_rgw_gc_log ${rgw_libs} radostest-cxx)
```

Comment:
> qa/workunits/rgw/test_rgw_gc_log.sh: line 3: ceph_test_rgw_gc_log: command not found

i think this part is failing because we're not installing ceph_test_rgw_gc_log with the test packages. adding this should fix it:
```cmake
install(TARGETS ceph_test_rgw_gc_log DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Reply:
> > qa/workunits/rgw/test_rgw_gc_log.sh: line 3: ceph_test_rgw_gc_log: command not found
> 
> i think this part is failing because we're not installing ceph_test_rgw_gc_log with the test packages. adding this should fix it:
> 
> ```cmake
> install(TARGETS ceph_test_rgw_gc_log DESTINATION ${CMAKE_INSTALL_BINDIR})
> ```

Fixed this.